src/tests/integration/mapping.test.ts

```diff
@@ -67,6 +67,45 @@ describe('/api', () => {
     const mappingData = await getMapping.json()
     expect(Object.keys(mappingData).length).toEqual(0)
   })
+  
+  it('checks if changes to the resource has been saved', async () => {
+    const subDomain = `testing${uuidv4()}`
+    const domain = 'integration'
+    const port = '3457'
+    const createMapping = await mappingAdapter('/', 'POST', {
+      domain, 
+      subDomain,
+      port
+    })
+    expect(createMapping.status).toEqual(200)
```

Comment:
no need for a check here. Therre's already a check for `createMapping` status. Its good practice to just focus on exactly whhat you are testing, whichhh is thhe update endpoint.

sorry for the typos, keyboards on thhe macbook is so messed up. Double chharracters, keys sometimes not registering, and a broken command key. Do not buy a macbook.... lol.